nvidia_triton/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - Nvidia Triton
+
+## 1.0.0 / 2023-10-11
```

Comment:
@iliakur aren't we supposed to have an "unreleased" section in the template file?

Reply:
iiuc this was removed from the integration template because marketplace/extras also used the template without using the `release make` command to replace "unreleased" header.

TBH I'm a bit confused about how we release new integrations now, independent of which system we use (single or multi file)...

Reply:
right but you changed everything, hence my question.

Reply:
Oh, I focused on the "unreleased" part and didn't see the big picture question, sorry.

I'm looking into this. Options so far:

1. add `no-changelog` to new integration PRs, keep changelogs as they are.
2. somehow make the "new integration template" support both our workflow and that of external contributors.
3. manually adjust the changelog coming from the template to fit our new system (changelog gets generated when we release).